pkg/planner/util/handle_cols.go

```diff
@@ -64,6 +66,10 @@ type HandleCols interface {
 	MemoryUsage() int64
 	// Clone clones the HandleCols.
 	Clone(newCtx *stmtctx.StatementContext) HandleCols
+	// IterColumns iterates the columns.
+	IterColumns() iter.Seq[*expression.Column]
+	// IterColumns2 iterates the columns.
+	IterColumns2() iter.Seq2[int, *expression.Column]
```

Comment:
Xx2 is not a good name.
Itercolumnvalue/itercolumnIdxValue

Reply:
And amao once said we should make interface simple as it should be. Can we merge it together distinguished with parameters？

Reply:
In fact, it is disgusting that golang gave itself a name of ```seq2```

<img width="540" alt="image" src="https://github.com/user-attachments/assets/33c01b4e-b73e-409c-a4f6-2e675db3576c" />